go-controller/pkg/ovn/gateway.go

```diff
@@ -1172,9 +1172,8 @@ func (gw *GatewayManager) syncNodeGateway(
 }
 
 func physNetName(netInfo util.NetInfo) string {
-	physnetName := types.PhysicalNetworkName
-	if netInfo.IsSecondary() {
-		physnetName = netInfo.GetNetworkName()
+	if netInfo.IsDefault() || netInfo.IsPrimaryNetwork() {
+		return types.PhysicalNetworkName
 	}
-	return physnetName
+	return netInfo.GetNetworkName()
```

Comment:
we are not supporting pod2Egress for secondary networks.. so for now support is only being added for primary and for that @dceara mentioned its enough we keep it as physnet... maybe even for 2ndary networks we only actually ever need physnet because our bridge is the same ultimately.. the OVN networks are N mapped to 1 physnet right?

Reply:
no, for secondary (localnet) networks you *can* have configurable bridges. The user is responsible for setting up the underlay network (e.g. create the OVS bridge, attach & configure interface to the bridge, vlan mumbo-jumbo, etc), and then they'll configure the mappings accordingly - something like (assuming kubernetes-nmstate is used): 

```yaml
    ovn:
      bridge-mappings:
      - localnet: secret-net
        bridge: mytinywinybr 
        state: present 
```

Reply:
@maiqueb do youve docs to setup that? Id like to do it myself some day.

Reply:
@martinkennelly 
https://docs.openshift.com/container-platform/4.16/networking/multiple_networks/configuring-additional-network.html#configuring-additional-network_configuration-additional-network-interface

and there are these upstream resources for it:
- https://github.com/maiqueb/fosdem2024-sdn?tab=readme-ov-file#localnet-scenario---accessing-the-physical-network
- https://maiqueb.github.io/blog/2023/11/23/openshift-microsegmentation.html#scenario

Reply:
all this need sto be added to ovn-kubernetes.io